src/core/numerics/time_integrator/euler_integrator.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef PHARE_CORE_NUMERICS_EULER_INTEGRATOR_HPP
+#define PHARE_CORE_NUMERICS_EULER_INTEGRATOR_HPP
+
+#include "initializer/data_provider.hpp"
+#include "core/numerics/time_integrator/euler.hpp"
+
+namespace PHARE::core
+{
+template<template<typename> typename FVMethod, typename MHDModel>
+class EulerIntegrator
```

Comment:
Maybe I don't remember, but reading this, it seems this class only wraps the other Euler without adding any extra feature and thus has no real purpose and the other Euler could the EulerIntegrator